packages/nextjs/app/builders/0xE451141fCE63EB38e85F08a991fC5878Ee6335b2/page.tsx

```diff
@@ -0,0 +1,181 @@
+import Image from "next/image";
+import Link from "next/link";
+import type { Metadata, NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+export const metadata: Metadata = {
+  title: "Parizval — Batch #21",
+  description: "Builder page for Batch #21",
+};
+
+const ADDRESS = "0xE451141fCE63EB38e85F08a991fC5878Ee6335b2";
+
+const ParizvalPage: NextPage = () => {
+  return (
+    <main className="min-h-screen bg-white dark:bg-black text-gray-800 dark:text-gray-300 font-mono p-6">
+      <div className="mx-auto max-w-6xl grid grid-cols-1 md:grid-cols-3 gap-8">
+        {/* Left Column */}
+        <section className="md:col-span-1 space-y-6">
+          <div className="rounded-2xl border border-cyan-500/30 shadow-lg shadow-cyan-500/10 p-6 bg-gray-100 dark:bg-gray-900/50 backdrop-blur-sm">
+            <div className="flex flex-col items-center text-center">
+              <Image
+                src="https://avatars.githubusercontent.com/u/48042530?v=4"
+                alt="avatar"
+                width={120}
+                height={120}
+                className="w-32 h-32 rounded-full border-2 border-cyan-400 shadow-lg shadow-cyan-500/50"
+              />
+              <h1 className="text-2xl font-bold text-cyan-600 dark:text-cyan-300 mt-4">Parizval</h1>
+              <p className="text-sm text-gray-500 dark:text-gray-400">Batch #21 — Builder</p>
+              <div className="mt-2">
+                <Address address={ADDRESS} />
+              </div>
+              <p className="mt-4 text-gray-600 dark:text-gray-400">DeFi Wizard</p>
+              <p className="mt-2 text-gray-600 dark:text-gray-400">
+                Learning Core Protocol Development and Pushing Ethereum Towards Interoperability.
+              </p>
+            </div>
+
+            <div className="mt-6">
+              <h2 className="text-sm uppercase tracking-wider text-cyan-500 dark:text-cyan-400 text-center">
+                Tech stack
+              </h2>
+              <div className="mt-2 flex flex-wrap gap-2 justify-center">
+                <span className="text-xs px-2 py-1 rounded-full border border-green-500/50 bg-green-100 dark:bg-green-900/30 text-green-700 dark:text-green-300">
+                  Solidity
+                </span>
+                <span className="text-xs px-2 py-1 rounded-full border border-green-500/50 bg-green-100 dark:bg-green-900/30 text-green-700 dark:text-green-300">
+                  Rust
+                </span>
+                <span className="text-xs px-2 py-1 rounded-full border border-green-500/50 bg-green-100 dark:bg-green-900/30 text-green-700 dark:text-green-300">
+                  Typescript
+                </span>
+                <span className="text-xs px-2 py-1 rounded-full border border-green-500/50 bg-green-100 dark:bg-green-900/30 text-green-700 dark:text-green-300">
+                  Python
+                </span>
```

Comment:
Consider mapping over a list of data instead of repeating the JSX for each one. It'll make the code cleaner and easier to maintain.